src/test/java/hudson/plugins/clover/CloverBuildActionTest.java

```diff
@@ -76,13 +76,16 @@ private void checkCloverReports(Run<?, ?> build, Job<?, ?> project) throws Excep
         try (JenkinsRule.WebClient wc = j.createWebClient()) {
```

Comment:
I think this test should handle a case for multiple Clover reports as well (rather in a separate method).